include/alpaka/queue/cuda_hip/QueueUniformCudaHipRt.hpp

```diff
@@ -85,6 +88,8 @@ namespace alpaka
 
         public:
             DevUniformCudaHipRt<TApi> const m_dev; //!< The device this queue is bound to.
+            core::CallbackThread c_thread;
```

Comment:
```suggestion
            core::CallbackThread m_thread;
```

Reply:
or something more descriptive, like `m_callbackThread`

Reply:
@tonydp03 please address this! `c_` is not a good prefix, since it is easily confused with a constant in Hungarian notation.

Reply:
yes, I modified the name in the class instead of here. My bad, sorry :)